SU2_CFD/src/fluid/CFluidScalar.cpp

```diff
@@ -48,14 +48,13 @@ CFluidScalar::CFluidScalar(su2double val_Cp, su2double val_gas_constant, su2doub
       n_species_mixture(config->GetnSpecies() + 1),
       Gas_Constant(val_gas_constant),
       Gamma(config->GetGamma()),
-      Pressure_Thermodynamic(value_pressure_operating),
```

Comment:
Is the value not constant throughout the simulation anymore?

Reply:
Hi Pedro, thank you so much for your comments. It is still constant but now it is a user input. I have deleted this constant variable because after comparing with the CIncIdealGas.cpp I realized that the Pressure there has the same meaning as the Pressure used in the CFluidScalar. So I think this Pressure_Thermodynamic is redundant.

Reply:
If the value is constant please keep using a `const su2double` variable, the other models should have done the same.

Reply:
Understood, I will keep the current implementation. Thank you so much for your comments!!